### ISSUE TITLE
Direct user to the profile tab vs the orders tab when clicking on it from the header  - desktop 

### ISSUE THREAD

**Issue by suehamre** [tastefully-simple/cornerstone#212]
### Expected behavior
User should be brought to the profile tab when logged in and clicking on profile in the header. 

### Actual behavior
User is brought to the orders tab when clicking on profile in the header on desktop. Mobile already works like this.
![image](https://user-images.githubusercontent.com/57115316/88852264-a6710f80-d1b3-11ea-9b65-d3384f799aaa.png)

### Steps to reproduce behavior
Login to an account on desktop and click on profile. 

NOTE: Chatted with Jen about this in slack. 
Added correct screen shot.

**Comment by wbyrnetx** [comment]
I did a little research on this. The link in the header is actually correct. It's pointing to `/account.php`, which is just the profile tab (what we want).

However... when the request is made to /account.php (profile home), BigCommerce is responding back with a 302 redirect response to `/account.php?action=order_status` ... which is why you end up on the "Orders" tab.
<img width="696" alt="Screen Shot 2020-07-30 at 9 44 31 AM" src="https://user-images.githubusercontent.com/4029452/88930608-b3d8d900-d249-11ea-8499-f3077cc25ba1.png">
<img width="556" alt="Screen Shot 2020-07-30 at 9 44 41 AM" src="https://user-images.githubusercontent.com/4029452/88930622-b63b3300-d249-11ea-81f2-444b3820ac67.png">

Since this redirect is coming from the BigCommerce backend, I believe we need to ask them what's going on with this.

I've created [TST-141](https://razoyo.atlassian.net/browse/TST-141) in our system to investigate. We will keep you posted.
